src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -21,7 +21,6 @@
     <Compile Include="JsonElementWriteTests.cs" />
     <Compile Include="JsonEncodedTextTests.cs" />
     <Compile Include="JsonGuidTestData.cs" />
-    <Compile Include="JsonNumberTestData.cs" />
```

Comment:
Why was this moved? Leave it as is.

Reply:
Yes, sorry, I thought it was created by me, but it's a part of JsonElement tests.